### PR TITLE
Enable asset store for Godot 3.0

### PR DESCRIPTION
**Pull request by Chaosus**
I enabled it and allow loads only 3.0 assets from site - This is simple fix in http request line, I wonder why nobody touches this...

fix for https://github.com/godotengine/godot/issues/7147

image

### CODE REVIEW

**Comment by Chaosus** [comment]
And why its printed ERROR in the description of each asset - its a subject of another issue, if I remove filter it loads 2.1 with that bug for each assets, my PR does nothing with it - it just enabled asset store and make it use 3.0 filter. Installation, searching worked as expected..

File: editor/plugins/asset_library_editor_plugin.cpp [hunk 1]
```diff
@@ -867,6 +867,8 @@ void EditorAssetLibrary::_search(int p_page) {
 	}
 	args += String() + "sort=" + sort_key[sort->get_selected()];
 
+	args += "&godot_version=3.0";
```
Comment (volzhs) [inline editor/plugins/asset_library_editor_plugin.cpp]: can we grab version number from somewhere, instead of hard code?

Comment (Chaosus) [reply editor/plugins/asset_library_editor_plugin.cpp]: I looking for possible solution...

File: editor/plugins/asset_library_editor_plugin.cpp [hunk 2]
```diff
@@ -1348,13 +1350,9 @@ EditorAssetLibrary::EditorAssetLibrary(bool p_templates_only) {
 	search_hb2->add_child(memnew(Label(TTR("Site:") + " ")));
 	repository = memnew(OptionButton);
 
-	// FIXME: Reenable me once GH-7147 is fixed.
-	/*
 	repository->add_item("godotengine.org");
 	repository->set_item_metadata(0, "https://godotengine.org/asset-library/api");
-	*/
-	repository->add_item("localhost");
-	repository->set_item_metadata(/*1*/ 0, "http://127.0.0.1/asset-library/api");
```
Comment (bojidar-bg) [inline editor/plugins/asset_library_editor_plugin.cpp]: You should leave localhost, it is useful when testing the assetlib (also, uncomment the `1` there).

Comment (Chaosus) [reply editor/plugins/asset_library_editor_plugin.cpp]: Ok

Comment (Chaosus) [reply editor/plugins/asset_library_editor_plugin.cpp]: Check ?

**Comment by Chaosus** [comment]
Ok, I squashed these commits into one

File: editor/plugins/asset_library_editor_plugin.cpp (hunk 2 above)
Comment (Chaosus) [reply editor/plugins/asset_library_editor_plugin.cpp]: Ah, you write uncommenting "1" ? Ok..

Comment (akien-mga) [reply editor/plugins/asset_library_editor_plugin.cpp]: It should not be commented out, it should be active, with id `1` instead of `0`.

Comment (Chaosus) [reply editor/plugins/asset_library_editor_plugin.cpp]: Ok

File: editor/plugins/asset_library_editor_plugin.cpp [hunk 3]
```diff
@@ -1348,13 +1352,14 @@ EditorAssetLibrary::EditorAssetLibrary(bool p_templates_only) {
...
+
+	/* reserved */
```
Comment (akien-mga) [inline editor/plugins/asset_library_editor_plugin.cpp]: Probably not needed, it doesn't say much and the scope of localhost is quite obvious by itself.

File: editor/plugins/asset_library_editor_plugin.cpp (hunk 2 above)
Comment (Chaosus) [reply editor/plugins/asset_library_editor_plugin.cpp]: repository->set_item_metadata(0, "https://godotengine.org/asset-library/api");

should also be 1 instead 0 ?

Comment (akien-mga) [reply editor/plugins/asset_library_editor_plugin.cpp]: It should be:
```
repository->add_item("godotengine.org");
repository->set_item_metadata(0, "https://godotengine.org/asset-library/api");
repository->add_item("localhost");
repository->set_item_metadata(1, "http://127.0.0.1/asset-library/api");
```

Comment (Chaosus) [reply editor/plugins/asset_library_editor_plugin.cpp]: Ah.. ok

**Review by akien-mga — APPROVED** [review]

**Pull request merged**